### ISSUE TITLE
Strong named assemblies for .NET 4.x?

### ISSUE THREAD

**Issue by Crono1981** [HugoRoss/Hafner.Compatibility.NullableReferenceTypeAttributes#1]
Hi,

First, I'd like to say thanks for taking the time to do this package.

Would you consider strong naming the assemblies for .NET 4.x targets? This is required for a lot of projects, mine included. 😇

Thank you so much!

**Comment by HugoRoss** [comment]
Yes, I definitely consider it - although my first priority now is to create
another package that provides the Dapper converters for Nullable value
types (nullable/non-nullable reference types work out of the box).

Am Sa., 19. Nov. 2022 um 00:23 Uhr schrieb David ***@***.***>:
